trans/TeXworks_sl.ts

```diff
@@ -879,7 +879,7 @@ Do you want to continue?</source>
         <location filename="../src/PrefsDialog.cpp" line="543"/>
         <source> (%1)</source>
         <extracomment>Country suffix for TeXworks translations (ex. &quot;Portuguese (Brazil)&quot;)</extracomment>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished"> (%1)</translation>
```

Comment:
I didn't quite understand what you want here / why is this even translatable.

Reply:
You are right, this should be changed